.pre-commit-config.yaml

```diff
@@ -0,0 +1,14 @@
+repos:
```

Comment:
did you mean to include this file? I assume this was an accidental commit

Reply:
whoops dropped.